src/app/models/feedback.py

```diff
@@ -0,0 +1,120 @@
+# import logging
+
+from sqlalchemy import func, text
+from sqlalchemy.engine import Result
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import aliased
+from sqlalchemy.sql.expression import select
+
+from src.app.views.response.feedback import Feedback as ResponseFeedback
+from src.app.views.response.feedback import FeedbackCount
+from src.core.database.models.feedback import DataModelPredictionFeedback as dbFeedback
+from src.core.database.models.player import Player as dbPlayer
+
+
+class Feedback:
+    def __init__(self, session: AsyncSession) -> None:
+        self.session = session
+        # self.logger = logging.getLogger("Feedback")
+
+    async def get_feedback(self, player_names: list[str]):
+        async with self.session:
+            # print(f"Player names: {player_names}")
+            feedback_voter: dbFeedback = aliased(dbFeedback, name="feedback_voter")
+            feedback_subject: dbFeedback = aliased(dbFeedback, name="feedback_subject")
+            player_name: dbPlayer = aliased(dbPlayer, name="player_name")
+
+            query = select(
+                [
+                    feedback_voter.vote,
+                    feedback_voter.prediction,
+                    feedback_voter.confidence,
+                    player_name.name,
+                    feedback_voter.feedback_text,
+                    feedback_voter.proposed_label,
+                ]
+            )
+            query = query.select_from(player_name)
+            query = query.join(
+                feedback_subject, feedback_subject.subject_id == player_name.id
+            )
+            query = query.join(
+                feedback_voter, feedback_voter.voter_id == player_name.id
+            )
+            query = query.where(
+                text("player_name.name IN :names").bindparams(
+                    names=player_names
+                )  # prevent sql injection
```

Comment:
i think it is better to not "raw dog" sql
```
query.where(dbPlayer.name.in_(player_names))
```

Reply:
this is a strategy to prevent sql injection using the bind method

Reply:
this also prevents sql injection and is a bit cleaner, using the orm
```
query.where(dbPlayer.name.in_(player_names))
```

Reply:
[refactor player_name alias to player_db, remove bindparams](https://github.com/Bot-detector/public-api/pull/25/commits/91bdf391df482e4511ec664118865f178383a9cc)